src/ceph_broker.py

```diff
@@ -1103,7 +1103,25 @@ def handle_create_cephfs_client(request, service):
         log(msg, level=ERROR)
         return {"exit-code": 1, "stderr": msg}
 
+    # Skip creation if the request has already been called
+    # This makes it a bit more compatible with older Ceph versions
+    # that throw when trying to authorize a user with the same
+    # capabilities that it currently has.
+    try:
+        cmd = ["microceph.ceph", "--id", service, "auth", "ls", "-f", "json"]
```

Comment:
can this not be done using the `get-or-create` behavior ? also I am not sure if i understand the problem correctly.

Reply:
ack, please refer to [this documentation](https://docs.ceph.com/en/reef/rados/operations/user-management/#adding-a-user)

Reply:
Applied the change